testdata/p4_16_errors_outputs/spec-issue1297-string-cat-err-1.p4-stderr

```diff
@@ -0,0 +1,9 @@
+spec-issue1297-string-cat-err-1.p4(5): [--Wwarn=mismatch] warning: 1w2: value does not fit in 1 bits
+    log(1w2
+        ^^^
+spec-issue1297-string-cat-err-1.p4(4): [--Werror=type-error] error: Please specify a width for the operand 4 of a concatenation
```

Comment:
Nit: This error message should probably be eclipsed by one on line 7. Ordering issue in type checking?

Reply:
I think it goes in postorder, but let me double check it -- the non-width literal should get type `int` so we should be able to handle this at the `++` level, not at the literal's leaf node.

Reply:
Indeed this was a bug in the concat case.